corehq/project_limits/rate_limiter.py

```diff
@@ -193,11 +200,12 @@ def map(self, math_func, other=None):
 
     def get_rate_limits(self):
         return [(rate_counter, limit) for limit, rate_counter in (
-            (self.per_week, week_rate_counter),
-            (self.per_day, day_rate_counter),
-            (self.per_hour, hour_rate_counter),
-            (self.per_minute, minute_rate_counter),
+            # order matters for returning the highest priority window
```

Comment:
this was the best way I saw for "sorting" the rate limits so that in the future, the order of iteration would be correct in terms of priority.

Reply:
I think this change does the opposite of what you want it to. Since these are checked in order in a loop, and the loop exits on the first one it finds where the limit is exceeded, you want to have week first and second last. Right?

Reply:
Ah sorry you are right, I mixed this one up. I'll leave a comment in there about order mattering.